content/news/2017-02-20-mapstruct-1_2_0_Beta1-is-out-with-lombok-support-and-direct-field-access.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Support for Lombok, direct field access and much more: MapStruct 1.2.0.Beta1 is out"
+author: Gunnar Morling
+date: "2017-02-20"
+tags: [release, news]
+---
+
+It' my pleasure to announce the first Beta release of MapStruct 1.2.
+
+The new release comes with a whole lot of new functionality, e.g.:
+
+* MapStruct can now be used together with Project Lombok
+* Support for Java 8 streams
+* Mappings based on public fields (i.e. you can write DTOs without getters and setters)
+* Automatic creation of sub-mapping methods
+* Mapping methods can take "pass-through" context parameters, e.g. for passing a locale to custom mapping methods
```

Comment:
maybe add the possibility to handle cycles here as well, as it's something that has been long awaited (in contrast to the locale, which we made up in the example 😁 )